src/main/java/seedu/tasklist/ui/UpcomingTaskListPanel.java

```diff
@@ -48,7 +48,7 @@ public UpcomingTaskListPanel(ObservableList<Task> taskList) {
         upcomingTaskListView.setCellFactory(listView -> new UpcomingTaskListViewCell());
     }
 
-    public static void getDueDate(Label label, Deadline deadline) {
+    public static void getDueDate(Label label, Time deadline) {
         LocalDateTime now = LocalDateTime.now();
         LocalDateTime due = LocalDateTime.parse(deadline.value, inputFormat);
         String formattedDue = due.format(DateTimeFormatter.ofPattern("dd MMM yyyy HH:mm"));
```

Comment:
Likewise for here. You can redirect this issue to @minhhhnguyen2000 if you wish XD